android/src/main/java/io/invertase/firebase/auth/RNFirebaseAuth.java

```diff
@@ -1177,6 +1179,31 @@ public void onComplete(@NonNull Task<ProviderQueryResult> task) {
       });
   }
 
+  /**
+   * Set the language code for the auth module
+   * @param appName
+   * @param code
+   */
+  @ReactMethod
+  public void setLanguageCode(String appName, String code) {
+    FirebaseApp firebaseApp = FirebaseApp.getInstance(appName);
+    FirebaseAuth firebaseAuth = FirebaseAuth.getInstance(firebaseApp);
+
+    firebaseAuth.setLanguageCode(code);
+  }
+
+  /**
+   * Use the device language
+   * @param appName
+   */
+  @ReactMethod
+  public void useDeviceLanguage(String appName) {
```

Comment:
@Salakar @chrisbianca should I leave the native ones here for the time being? JS currently flags as unsupported due to Firebase issue, but no harm in them being here?